presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcColumnHandle.java

```diff
@@ -32,18 +33,21 @@
     private final JdbcTypeHandle jdbcTypeHandle;
     private final Type columnType;
     private final boolean nullable;
+    private final Optional<String> comment;
 
     @JsonCreator
     public JdbcColumnHandle(
             @JsonProperty("columnName") String columnName,
             @JsonProperty("jdbcTypeHandle") JdbcTypeHandle jdbcTypeHandle,
             @JsonProperty("columnType") Type columnType,
-            @JsonProperty("nullable") boolean nullable)
+            @JsonProperty("nullable") boolean nullable,
+            @JsonProperty("comment") Optional<String> comment)
```

Comment:
Alternatively https://github.com/prestosql/presto/pull/1898

Reply:
Thanks! Let me update this PR after it was merged.